crates/dbt-deps/src/private_package.rs

```diff
@@ -116,6 +116,16 @@ mod tests {
     use super::*;
 
     #[test]
+    fn tests() {
+        // Run the tests sequentially to avoid flakyness when running in parallel due to
```

Comment:
I feel it would be nicer to change existing tests to `assert` that the variable is not set. No test should ever set (that) variable. Those that have the variable set should change that in that own environment and not impose restrictions on the tests we have.

Reply:
if that is the case, then this PR is fine
but as long as we have `allow` `disallowed_methods` it is not good